misc_scripts/backfill_drugs.rb

```diff
@@ -0,0 +1,40 @@
+drugs_to_delete = []
+Drug.where(ncit_id: nil).each do |drug|
+  uri = URI.parse('http://data.bioontology.org/search').tap do |u|
+    u.query = URI.encode_www_form({ontologies: 'NCIT', q: drug.name, require_exact_match: true, apikey: ENV['BIOONTOLOGY_API_KEY']})
+  end
+  res = Net::HTTP.get_response(uri)
+  raise StandardError.new("Request Failed!") unless res.code == '200'
+  response = JSON.parse(res.body)
+  if response['totalCount'] == 1
+    match = response['collection'].first
+  elsif response['totalCount'] > 1
+    response['collection'].each do |r|
+      if r['prefLabel'].downcase == drug.name.downcase
+        match = r
+        break
+      end
+    end
+  else
+    next
+  end
+  if not match.nil?
```

Comment:
Same comment about `not` here, also could this just be `if match` or `if match.present?` instead of the double negative?